src/mqtt/MQTT.cpp

```diff
@@ -62,12 +62,17 @@ MQTT::MQTT() : concurrency::OSThread("mqtt"), pubSub(mqttClient)
 void MQTT::reconnect()
 {
     // pubSub.setServer("devsrv.ezdevice.net", 1883); or 192.168.10.188
-    const char *serverAddr = "mqtt.meshtastic.org"; // default hostname
+    const char *serverAddr = "mqtt.meshtastic.org:1883"; // default hostname
 
     if (*radioConfig.preferences.mqtt_server)
         serverAddr = radioConfig.preferences.mqtt_server; // Override the default
 
-    pubSub.setServer(serverAddr, 1883);
+    String server = String(serverAddr);  
+    int delimIndex = server.indexOf(':'); 
```

Comment:
Does this actually build?  I am not aware of indexOf or substring being in the C++ standard library.